locations/spiders/dominos_pizza.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please only include one spider in a pull request. Remove this spider from this pull request.